hebiros/include/hebiros/hebiros_group.h

```diff
@@ -1,30 +1,24 @@
-#ifndef HEBIROS_GROUP_H
-#define HEBIROS_GROUP_H
+#pragma once
 
 #include "ros/ros.h"
 #include "sensor_msgs/JointState.h"
 
 #include "hebiros/FeedbackMsg.h"
 
-
+// Base class for physical and simulated groups of modules.
 class HebirosGroup {
 
   public:
 
-    static std::map<std::string, std::shared_ptr<HebirosGroup>> groups;
+    HebirosGroup();
+    virtual ~HebirosGroup() {};
 
-    std::string name;
     int size;
     std::map<std::string, std::string> joint_full_names;
     std::map<std::string, int> joints;
     sensor_msgs::JointState joint_state_msg;
     hebiros::FeedbackMsg feedback_msg;
 
-    HebirosGroup(std::string name);
-    static std::shared_ptr<HebirosGroup> getGroup(std::string name);
-    static void removeGroup(std::string name);
-    static bool findGroup(std::string name);
-
+    virtual void setFeedbackFrequencyHz(float frequency);
```

Comment:
Do we think the unit suffix is really necessary? I could go either way here, but I would prefer terseness.